locales/en/index.yml

```
@@ -2458,7 +2458,7 @@ features:
         title: "TMP"
         subtitle: "TMP"
       notOperational:
-        title: "TMP"
+        title: "The european COVID certificate it actually inactive"
```

```suggestion
        title: "The european COVID certificate is not currently operational"
```
I guess actually doesn't mean "at this moment"